File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int k=0;
        for(int i=0;i<nums.length;i++){
            nums[k]=nums[i];
            if(i<nums.length-1 && nums[k]!=nums[i+1]) k++;
            // else if(nums[k])
        }
        return k+1;
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let val=nums[0];
    let j=1;
    for(let i=1;i<nums.length;i++){
        if(val===nums[i]){
            continue;
        }else{
             nums[j]=nums[i];
             val=nums[j];
             j++;
        }
    }
    return j;
};